Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/ExposureWindowRiskCalculationConfigMapper.kt

```diff
@@ -34,10 +41,24 @@ class ExposureWindowRiskCalculationConfigMapper @Inject constructor() :
             normalizedTimePerExposureWindowToRiskLevelMapping = riskCalculationParameters
                 .normalizedTimePerEWToRiskLevelMappingList,
             normalizedTimePerDayToRiskLevelMappingList = riskCalculationParameters
-                .normalizedTimePerDayToRiskLevelMappingList
+                .normalizedTimePerDayToRiskLevelMappingList,
+            diagnosisKeysDataMapping = rawConfig.diagnosisKeysDataMapping()
         )
     }
 
+    private fun AppConfigAndroid.ApplicationConfigurationAndroid.diagnosisKeysDataMapping():
```

Comment:
Why are we creating the mapping, and then taking the mapping, from protobuf class to protobuf class?
Could we use a data class here, to keep the protobuf classes hidden?

Reply:
the second class is not a protobuf but the internal enf config used by google, we could also implement a data class here, i oriented the initial implementation against the old mapRiskScoreToExposureConfiguration https://github.com/corona-warn-app/cwa-app-android/pull/1677/files#diff-0b01d7464a4e38df4e564db97b132516563e7b4ff1d004f4b6b0110d193f79edL67